samples/XamlIslandSample/Microsoft.Toolkit.Wpf.UI.XamlHost/Microsoft.Toolkit.Wpf.UI.XamlHost.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
+    <RootNamespace>Microsoft.Toolkit.Wpf.UI.XamlHost</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Wpf.UI.XamlHost</AssemblyName>
+
+    <Title>Windows Community Toolkit XAMLHost for WPF</Title>
+    <Description>This library provides XAML islands helpers for WPF. It is part of the Windows Community Toolkit.</Description>
```

Comment:
Is it really necessary to duplicate this code here rather than just pulling the toolkit down from NuGet?

Reply:
Problem is that all official preview builds that support .NET Core 3 only work on Windows insider builds. On RS5 everything builds and starts but crashes with TypeLoadException when the app window is moved. I had a PR over there but didn't get any response. I will change to the latest preview package from myget and update the readme